src/Microsoft.Health.Fhir.Web/appsettings.json

```diff
@@ -25,7 +25,11 @@
         "Host": null,
         "Key": null,
         "DatabaseId": "health",
-        "CollectionId": "fhir",
+        "InitialDatabaseThroughput": 400,  
```

Comment:
Should we considerr separating the setting for FHIR and the CP? I think it has a few advantages for doing so:

1. It allows different persistent layer to be used so the right technology/price could be picked.
2. It allows different connection to be used. For example, if I am hosting 2 FHIR servers but want to share the CP, i could use 2 separate Cosmos DB account for the FHIR but share the same Cosmos DB account for managing these two servers so that the roles could be shared between the two.

These would be some redundency in terms of settings but it gives us the flexibility?

Reply:
I've refactored to have the collection specific settings in the `FhirServer` section and a new `ControlPlane` section.